src/bms/parse/header.rs

```diff
@@ -34,6 +34,66 @@ pub struct Bmp {
     pub transparent_color: Argb,
 }
 
+/// A definition for #@BGA command.
+#[derive(Debug, Clone, PartialEq)]
+pub struct AtBgaDef {
+    /// The object ID.
+    pub id: ObjId,
+    /// The source BMP object ID.
+    pub source_bmp: ObjId,
+    /// The top-left position for trimming.
+    pub trim_top_left: (i16, i16),
+    /// The size for trimming.
+    pub trim_size: (u16, u16),
+    /// The draw point position.
+    pub draw_point: (i16, i16),
```

Comment:
You should denote what the element of these tuples represents. They can't be read from these doc comments. Please do the same for `BgaDef`.